changelogs/fragments/9419-inventory-cobbler-extra-vars.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - cobbler inventory plugin - request full rendered variables for Cobbler system, allow end user to specify extra groups, should not change currently expected behavior (https://github.com/ansible-collections/community.general/issues/9419, https://github.com/ansible-collections/community.general/pull/9420).
```

The PR number changed:
```suggestion
  - cobbler inventory plugin - request full rendered variables for Cobbler system, allow end user to specify extra groups, should not change currently expected behavior (https://github.com/ansible-collections/community.general/issues/9419, https://github.com/ansible-collections/community.general/pull/9975).
```